app/src/main/java/org/grapheneos/pdfviewer/PdfViewer.java

```diff
@@ -213,6 +215,7 @@ public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceReque
                     headers.put("Content-Security-Policy", CONTENT_SECURITY_POLICY);
                     headers.put("Permissions-Policy", PERMISSIONS_POLICY);
                     headers.put("X-Content-Type-Options", "nosniff");
+                    headers.put("Referrer-Policy", "no-referrer");
```

Comment:
I don't really see much point in adding this but we could do it if there's some reason for it.

Reply:
Referrer-Policy isn't useful in this case for localhost so I guess maybe it also doesn't matter if it exists or not? The default is `strict-origin-when-cross-origin`.

Can be removed as I'm not aware of it providing any security benefits for a localhost PDF viewer.

Reply:
Note it's not really using localhost but rather it just uses localhost as a placeholder and then intercepts all requests to localhost to provide them in the app code.

Reply:
Ah okay. I knew there was something special when I saw `https://` infront of localhost.

I can remove Referrer-Policy as then there's nothing beneficial if there isn't an actual " web server".

Reply:
It has no network access so it can't actually access localhost. The method you're changing is the `shouldInterceptRequest` override which substitutes the requests to https://localhost/ with different content.